rust/headless-client/src/dns_control/windows.rs

```diff
@@ -82,49 +78,50 @@ pub(crate) fn system_resolvers() -> Result<Vec<IpAddr>> {
     Ok(resolvers)
 }
 
+// Our NRPT rule should always live at
+// `Computer\HKEY_LOCAL_MACHINE\SYSTEM\CurrentControlSet\Services\Dnscache\Parameters\DnsPolicyConfig\{6C0507CB-C884-4A78-BC55-0ACEE21227F6}`
+const NRPT_REG_KEY: &str = "{6C0507CB-C884-4A78-BC55-0ACEE21227F6}";
```

Comment:
How was this determined? Do we get to pick it?

Reply:
It was chosenly randomly at dev time, we do get to pick it. The previous method with Powershell assigned a random one at runtime, so this is actually much nicer, that's why I was able to fix that DNS leak.

I added the first sentence as a comment